tests/src/e2e-smoke.spec.ts

```diff
@@ -30,13 +20,16 @@ beforeAll(async () => {
         height: 700,
       },
     },
-  });
+  };
+
+  pdRunner = new PodmanDesktopRunner('.', properties);
```

Comment:
I would expect
  const env: { [key: string]: string } = Object.assign({}, process.env as { [key: string]: string });
  env.PODMAN_DESKTOP_HOME_DIR = 'tests/output/podman-desktop';

+ height, width, record of videos, etc be part of the PodmanDesktopRunner

like nothing to setup, it just creates for me everything based on my 'id' (in that case for example 'smoke') (or it creates an id automatically)

with that id, it creates folder for video recording, screenshot, configuration folders, etc.

only if I want something different I could provide customization

And the same, probably start could return the page

```
podmanDesktopRunner = new PodmanDesktopRunner('smoke');
const page = await podmanDesktopRunner.start();
```

Reply:
sounds great!